modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosControllerEpics.scala

```diff
@@ -44,10 +44,10 @@ object GmosControllerEpics extends GmosSouthController {
 
   implicit val ampGainSettingEncoder: EncodeEpicsValue[AmpGainSetting, String] = EncodeEpicsValue(_.value.toString)
 
-  implicit val ampCountEncoder: EncodeEpicsValue[AmpCount, Int] = EncodeEpicsValue {
-    case AmpCount.THREE  => 3
-    case AmpCount.SIX    => 6
-    case AmpCount.TWELVE => 12
+  implicit val ampCountEncoder: EncodeEpicsValue[AmpCount, String] = EncodeEpicsValue {
+    case AmpCount.THREE  => "BEST"
+    case AmpCount.SIX    => "ALL"
+    case AmpCount.TWELVE => "SECONDARY"
```

Comment:
Contents of gmosAmpCount.lut:
#OT amp count	GMOS amp mode
Six				BEST
Twelve			ALL